aten/src/ATen/native/cuda/LogAddExpKernel.cu

```diff
@@ -13,20 +14,15 @@
 namespace at::native {
 
 void logaddexp_kernel_cuda(TensorIteratorBase& iter) {
-  AT_DISPATCH_FLOATING_TYPES_AND2(
+  AT_DISPATCH_FLOATING_AND_COMPLEX_TYPES_AND2(
```

Comment:
for enabling pw operations for complex we use jiterator, to not increase compilation time. Please take a look at other complex ops, the pattern is pretty simple

Reply:
Thanks @ngimel. From the other functions, I see most (if not all) of them uses the macro `#if AT_USE_JITERATOR()` and `#else`, which means they provide 2 implementations: one with jiterator and one without. Do I need to include the implementation without jiterator as well?

Reply:
That's a good question, perhaps all the CUDA versions we support now already have support for the nvrtc API we use @ngimel ?

Reply:
Initially, ROCM didn't support jiterated versions. Also, we've noticed that on windows jiterated versions inexplicably produced wrong results, at least for reductions, and no one was in the mood for debugging. But now it looks like it is just defined to `true`, so perhaps we don't need regular impl? We probably can put `static_assert` in the #else part to make sure it's fine.